src/app/Dashboard/styles/dashboard.scss

```diff
@@ -0,0 +1,146 @@
+.wrapper {
+  display: flex;
+  justify-content: space-around;
+}
+
+.main-content {
+  display: flex;
+  position: absolute;
+  top: 0;
+  z-index: 1;
+  width: 90%;
+}
+
+.sub-content {
+  flex-grow: 100;
+  font-family: 'DIN Pro';
+  padding-top: rem(23px);
+}
+
+.user-dashboard {
+  margin-top: rem(27px);
+}
+.user-dashboard__name {
```

Comment:
Add a space here? do we have eslint on our project?

Reply:
Yes, we do. But I do not think it's linting SCSS files. I sure will add the space and do the same to other SCSS files.